applications/forms.py

```diff
@@ -55,3 +55,78 @@ class Meta:
             "training_passed_at",
         ]
         model = ResearcherRegistration
+
+
+class ResearcherRegistrationSubmissionForm(ResearcherRegistrationPageForm):
+    """
+    Submission form for Researchers
+
+    When submitting an application we want to do extra validation of the
+    fields, however researchers are separate to pages so we take the form used
+    to create/edit researchers and add cross-field validation to it.
```

Comment:
> however researchers are separate to pages

I don't really understand what this has got to do with pages.  Is it the case that we don't want to do any validation when a user first adds researcher details, but we do want to do validation when they submit the form?

Reply:
TLDR: yes

The researcher views (add&edit) are part of the wizard flow but not part of the Wizard[Page] parts of the code base, they're specific views.  However, we want mirror the flow of relaxed validation on those pages with stricter validation on submission.  So the submission form marks some fields as required and does cross-field validation to enforce that stricter validation.

I've added some clarifying bits to the docstring.